src/include/duckdb/main/query_profiler.hpp

```diff
@@ -144,8 +145,10 @@ struct QueryMetrics {
 	Profiler attach_replay_wal_latency;
 	//! The timer for running checkpoints.
 	Profiler checkpoint_latency;
-	//! The timer for the WAL writes during COMMIT.
-	Profiler commit_write_wal_latency;
+	//! The timer for the COMMIT time during WAL writes.
```

Comment:
Then we can rephrase this: `The timer for committing the transaction-local storage`.